my-app/src/components/messageList.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import Message from './message';
+
+class MessageList extends Component {
+
+  state = {
+    newMessage: "",
+    messagesList: [
+      {
+      userName: 'Ralf',
+      timePosted: Date.now(),
+      myMessage: false,
+      message: 'here is someone\'s message'
+    },
+      {
+        userName: 'Caroline',
+        timePosted: Date.now(),
+        myMessage: true,
```

Comment:
how to tell if a message is yours?

you need the `username`

you'll have a string like this:

```
"someuser|1553699775343|hello you"
const [username, timestamp, message] = message.split('|')
```

username === username ? align left : align right

Reply:
Leave it untill login is complited